### ISSUE TITLE
Analytics not rendering correctly

### ISSUE THREAD

**Issue by stuartshields** [facebook/facebook-instant-articles-sdk-php#27]
Hi,

So if you do the following as per the Facebook SDK found here: https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/docs/QuickStart.md#elements you'll get errors:

``` php
 ->addChild(
            Analytics::create()
                ->withHTML(
                    <h1>Some custom code</h1>
                    <script>alert("test");</script>
                )
        )
```

However if you do the following it works like a charm, I think the Facebook SDK documentation needs to be updated to reflect this:

``` php
$document = new \DOMDocument();
		$fragment = $document->createDocumentFragment();
		$valid_html = @$fragment->appendXML( "<script>". $analytics_code ."</script>" );
		if ( $valid_html ) {
			$article->addChild(
				\Facebook\InstantArticles\Elements\Analytics::create()
						->withHTML(
							$fragment
						)
				);
		}
```

**Comment by m4olivei** [comment]
Yeah I ran into this yesterday as well.  Agreed.

**Comment by m4olivei** [comment]
Or even better, if the SDK could do the DocumentFragment stuff itself, that would be great.

**Comment by everton-rosario** [comment]
Hello @stuartshields  and @m4olivei, thank you for reporting the issue.
Unfortunately we ran into the same issues as you guys, fixed the test case, but the documentation became outdated.

Ill push the fix on doc and keep this issue open to solve this inside the "withHTML" method. So developers wont run into issues like this.

Thank you.

Once we have the final solution for this, Ill close the issue.

**Comment by everton-rosario** [comment]
Fixed on #124
